bin/browser-tests

```diff
@@ -75,6 +81,40 @@ if ($vagrant) {
     exit;
 }
 
+BEGIN {
+    # setenv.pl above unloads File:: modules but we need them
+    use Cwd qw(abs_path);
+    use File::Path qw(remove_tree);
+}
+
+# Get a path for nyc
+my $nyc = 'nyc';
+$nyc = abs_path("$wsl/../../../\@cypress/code-coverage/node_modules/nyc/bin/nyc.js") if $wsl;
```

Comment:
this gets installed as `nyc.js` on my mac.

I also had to add the location to the path which was `/Users/$username/node_modules/nyc/bin`

Reply:
It will be "nyc" in your `/Users/$username/node_modules/.bin` I think?

Did you install "nyc" yourself or only the code-coverage package?

On mine it is in `node_modules/@cypress/code-coverage/node_modules/.bin/nyc` and corresponding `node_modules/@cypress/code-coverage/node_modules/nyc/bin/nyc.js`

So if I know where cypress is installed I could do the relative path from that which should be `../../../@cypress/code-coverage/node_modules/.bin/nyc`, but do I know that with any certainty? Hmm.

Reply:
Right, of course you have a hidden bin directory. That totally makes sense.

Anyway, it is indeed in `/Users/$username/node_modules/.bin` so adding that to my path works fine. It's not in `node_modules/@cypress/code-coverage/node_modules/.bin/` though.

I've uninistalled nyc and the code coverage module and then re-installed the code-coverage module to get this as the end result FWIW.

Reply:
Okay, just for the record, here's the final skinny as I understand it.

Facts:
* All I want is a path to run nyc.
* The code already assumes (for non-WSL) cypress is on PATH, and you specify a path to cypress with WSL.

Where is nyc installed:
* If you install cypress and code-coverage with npm locally (no `-g` flag), with nothing else installed, then npm will install them in any node_modules directory it finds, up to your homedir. In that node_modules directory, cypress and nyc can be run from `node_modules/.bin`.
* However, if e.g. you already had an older version of nyc manually installed, then that version would be the one in `node_modules/.bin` and the code-coverage packaged version would be in `@cypress/code-coverage/node_modules/.bin`!
* If you install cypress and code-coverage with npm globally (with `-g` flag), then npm will install them in your "prefix" (probably /usr/local/lib/node_modules or AppData\npm\node_modules on Windows, or ~/.npm/node_modules for me because I have set that somewhere). It creates *top-level* executables in /usr/local/bin on Unix, or AppData\npm\ directly on Windows - so "nyc" doesn't get a executable as it's a dependency, all I have there is cypress. nyc can be found only as nyc.js in `@cypress/code-coverage/node_modules/nyc/bin`

Possible locations therefore of nyc or nyc.js:
* `$(npm prefix)/node_modules/@cypress/code-coverage/node_modules/.bin/nyc`
* `$(npm prefix)/node_modules/.bin/nyc`
* `$(npm prefix -g)/node_modules/@cypress/code-coverage/node_modules/.bin/nyc`
* `$(npm prefix -g)/[bin/]nyc` (if you manually install nyc globally)

Possible locations of cypress, which you must have added to your path to get this to run previously anyway (or be using WSL like me):
* `$(npm prefix)/node_modules/.bin/cypress`
* `$(npm prefix -g)/[bin/]cypress`

I think easiest would be to ask the user to install nyc manually (globally or not), and then assume the right dir is on their PATH and then this script doesn't have to care.

Reply:
Agreed. I think if you are running a code coverage tool you should be tech savy enough to include node modules in your path.